cmd/backup-manager/app/util/util.go

```diff
@@ -389,13 +389,23 @@ func GetCommitTsFromBRMetaData(provider v1alpha1.StorageProvider) (uint64, error
 }
 
 // ConstructArgs constructs the rclone args
-func ConstructArgs(conf string, opts []string, command, source, dest string) []string {
+func ConstructArgs(conf string, opts []string, command, source, dest string, rcloneLog bool) []string {
 	var args []string
+	defaultLog := true
 	if conf != "" {
 		args = append(args, conf)
 	}
 	if len(opts) > 0 {
-		args = append(args, opts...)
+		for _, opt := range opts {
+			if rcloneLog == false && strings.HasPrefix(opt, "-v") {
```

Comment:
```suggestion
			if !rcloneLog && strings.HasPrefix(opt, "-v") {
```

Reply:
if seems here if `rcloneLog == false` and `opt == "-v"`, `defaultLog` will be `false`, but actually we should enable verbose log in this situation, which means `defaultLog` should be `true`?

and why use `strings.HasPrefix(opt, "-v")` but not `opt == "-v"`?

Reply:
1. `rcloneLog == false` is set only when command is `rclone ls` in fun `getBackupSize` in `cmd/backup-manager/app/export/export.go`. In this situation, we should filter any flag that could output logging, otherwise the output of `rclone ls` will not be size of data, and fun `getBackupSize` will fail.
2. `rclone` [logging](https://rclone.org/docs/#logging) could be setted such as `-v`, `-vv`, `-v 4`, `-v=4`, etc.

Reply:
> rcloneLog == false is setted only when command is rclone ls in fun getBackupSize in cmd/backup-manager/app/export/export.go. In this situation, we should filter any flag that could output logging, otherwise the output of rclone ls will not be size of data, and fun getBackupSize will fail.

If so, `rcloneLog` is not a proper parameter name, which actually represents such a complicated combination case. And note that `ConstructArgs()` has nothing todo with `rclone`, this is also weird, because I only see this function be called with "rclone" command.
If this function should be kept generic (not specific to "rclone"), then I think a more proper parameter would be `allowVerboseOutput`.
I also suggest a comment here, like `// if command don't allow verbose output, and user proveded args contains items starting with -v or --verbose, we should not add them to the final arguments`.

Reply:
I suggest rethinking about the design and purpose of `ConstructArgs()`, and answer the following questions:
1. is this specific to "rclone"?
2. if yes, we should rename this function.
3. if not, why is the verbose logic so specific to "rclone"? like `-v / -vv / -v=4` are all captured by `strings.HasPrefix()`?

Reply:
@dragonly @BinChenn Currently, `ConstructArgs()` is used for rclone only, I think it's OK to rename this function and rename the parameter name from `rcloneLog` to `verboseLog`, WDYT?

Reply:
@dragonly  @DanielZhangQD Yes, `ConstructArgs()` is used in rclone only, i will rename fun and logging parameters.

Reply:
Then LGTM